packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
+## 2.7.5-dev (Aug 29, 2023)
```

Comment:
you might need to redo the CHANGELOGs, switch from 2.7.5 to 2.8

Reply:
We can change when we officially release it